src/olympia/addons/migrations/0040_auto_20221011_1104.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.16 on 2022-10-11 11:04
+
+from django.db import migrations
+from olympia.constants.base import _ADDON_SITE_PERMISSION
+
+
+def delete_site_permissions(apps, schema_editor):
+    Addon = apps.get_model('addons', 'Addon')
+
+    # Hard delete the now obsolete site permission add-ons.
+    Addon.unfiltered.filter(type=_ADDON_SITE_PERMISSION).delete()
```

Comment:
Aren't there files, etc, that need to be tidied up too?

Reply:
Yeah... I'm not too worried about a few < 400 bytes files though. I figured the main issue was ensuring the database didn't contain anything the code couldn't handle properly anymore...

Reply:
Is there a plan for tidying up those files manually somehow, or are we okay with ignoring them?  I would have thought it'd be considerably harder to identify which are the obsolete files once we've nuked all the database references to them.

Reply:
We've had obsolete files on the filesystem before, I think it's fine to ignore them since they are so small and there are so few of them. We could technically identify them from their name (`site_permissions_for_*`) if we really need to find them and remove them later.